src/plz/plz.go

```diff
@@ -45,13 +46,13 @@ func Run(targets, preTargets []core.BuildLabel, state *core.BuildState, config *
 	wg.Add(config.Please.NumThreads + config.Remote.NumExecutors)
 	for i := 0; i < config.Please.NumThreads; i++ {
 		go func(tid int) {
-			doTasks(tid, state, parses, builds, tests, arch, false)
+			doTasks(ctx, tid, state, parses, builds, tests, arch, false)
```

Comment:
I'm not 100% sure about this... should these be a derived context at this point (since this is where it shards out to multiple parallel processes?)

Reply:
That sounds reasonable. What's a good way of doing that? None of the big four methods seem like what I want (which is just "give me a child context")

Reply:
hmmm good point - I'm not sure how opentracing would use the contexts to identify which thread things are on. Maybe you'd use a `ctx.WithValue` to attach something that identifies them? guess we can leave for now tho